daffodil-runtime1/src/main/scala/org/apache/daffodil/processors/unparsers/UState.scala

```diff
@@ -397,34 +397,31 @@ final class UStateForSuspension(
   override def withUnparserDataInputStream = mainUState.withUnparserDataInputStream
   override def withByteArrayOutputStream = mainUState.withByteArrayOutputStream
 
+  // $COVERAGE-OFF$
   override def advance: Boolean = die
   override def advanceAccessor: InfosetAccessor = die
   override def inspect: Boolean = die
   override def inspectAccessor: InfosetAccessor = die
-  override def fini: Unit = {
-    //do nothing
-  }
+  override def fini: Unit = die
   override def inspectOrError = die
   override def advanceOrError = die
   override def isInspectArrayEnd = die
-
   override def currentInfosetNodeStack = die
-  override def currentInfosetNodeMaybe = Maybe(currentInfosetNode)
-
   override def arrayIndexStack = die
   override def moveOverOneArrayIndexOnly() = die
-  override def arrayPos = occursIndex
-
   override def groupIndexStack = die
   override def moveOverOneGroupIndexOnly() = die
-  override def groupPos = 0 // was die, but this is called when copying state during debugging
-
   override def childIndexStack = die
   override def moveOverOneElementChildOnly() = die
-  override def childPos = 0 // was die, but this is called when copying state during debugging.
-
   override def pushDelimiters(node: DelimiterStackUnparseNode) = die
   override def popDelimiters() = die
+  // $COVERAGE-ON$
+
+  override def groupPos = 0 // was die, but this is called when copying state during debugging
+  override def currentInfosetNodeMaybe = Maybe(currentInfosetNode)
+  override def arrayPos = occursIndex
+  override def childPos = 0 // was die, but this is called when copying state during debugging.
+
```

Comment:
I looked at why groupPos and childPos's lines (420, 423) still aren't covered by tests (their original locations were inside the $COVERAGE-OFF$ block).  My best guess is that we don't have a test forcing the interactive debugger to copy unparser state.  I'll still give you a +1.